pints/_mcmc/__init__.py

```diff
@@ -1022,6 +1107,15 @@ def set_max_iterations(self, iterations=10000):
                     'Maximum number of iterations cannot be negative.')
         self._max_iterations = iterations
 
+    def set_max_initialisation_tries(self, max_initialisation_tries):
+        """
+        Sets number of attempts to initialise samplers by sampling.
```

Comment:
I'm a bit worried how this will look to a first (or 10th?) time user. Certainly needs a much clearer docstring. The current one assumes everyone's already aware that this is something the controller will do.

Reply:
Plus it doesn't always do it. So needs some condition :D

Reply:
Even a case to be made here for keeping this fixed for the controller, to avoid making it too clunky / adding too many bells & whistles?

Reply:
We could even do this:

1. Write a public `pints.sample_initial_point` method that does the sampling.
2. Have the controller use it with some fixed sensible max
3. Let the user who wants to customise things generate their own x0 _using this method_, so that they can pass in all the arguments and customisation they like, without cluttering the controller interface